Training25/Program.cs

```diff
@@ -3,12 +3,35 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T12 branch.
 // ------------------------------------------------------------------------------------------------
-namespace Training25 {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+using static System.Console;
+
+namespace Training25;
+
+internal class Program {
+   static void Main () {
+      string input;
+      Write ("Enter the string representation of the votes: ");
+      while (true) {
+         input = ReadLine ()?.ToLower () ?? "";
+         if (string.IsNullOrEmpty (input) || !input.All (Char.IsLetter)) {

```

Comment:
Follow `char.IsLetter` for uniformity. Also, code could be simplified as suggested below.
```suggestion
      while (true) {
         input = ReadLine ()?.ToLower () ?? "";
         if (!string.IsNullOrEmpty (input) && input.All (char.IsLetter)) break;
         Write ("Invalid input. Please enter a string containing only letters: ");
      }
```